lib/simp/packer/files/configs/centos6/simp_conf.yaml

```diff
@@ -0,0 +1,224 @@
+# The Base Distinguished Name of the LDAP server.
+simp_options::ldap::base_dn: dc=release,dc=me
+
+# === simp_options::ldap::bind_hash ===
+# The salted LDAP Bind password hash.
+simp_options::ldap::bind_hash: "{SSHA}ebr350mDOaiY3xzAHJPSxcGkhzvytOGB"
+
+# === simp_options::ldap::bind_pw ===
+# The LDAP Bind password.
+simp_options::ldap::bind_pw: hn3*dbV1tThnmav-ozA*LX1ee3d8U1GG
+
+# === simp_options::ldap::sync_hash ===
+#
+simp_options::ldap::sync_hash: "{SSHA}8+be6ONJ0gJpU5FULL24eTJysnrNXqkt"
+
+# === simp_options::ldap::sync_pw ===
+# The LDAP Sync password.
+simp_options::ldap::sync_pw: WhEybKTZdX5uqwbyxXOx8Ro2qGi9L+.U
+
+# === simp_options::ntpd::servers ===
+# Your network's NTP time servers.
+# A consistent time source is critical to your systems' security.
+# DO NOT run multiple production systems using individual hardware clocks!
+# For many networks, the default gateway (192.168.121.1) provides an NTP server.
+simp_options::ntpd::servers:
+- 192.168.121.1
+
+# === simp_options::puppet::ca ===
+# The Puppet Certificate Authority.
+simp_options::puppet::ca: puppet66.release.me
+
+# === simp_options::puppet::ca_port ===
+# The port on which the Puppet Certificate Authority will listen
+# (8141 by default).
+simp_options::puppet::ca_port: 8141
+
+# === simp_options::puppet::server ===
+# The Hostname or FQDN of the Puppet server.
+simp_options::puppet::server: puppet66.release.me
+
+# === simp_options::syslog::log_servers ===
+# The log server(s) to receive forwarded logs.
+#
+# No log forwarding is enabled when this list is empty.  Only use hostnames
+# here if at all possible.
+simp_options::syslog::log_servers: []
+
+# === simp_options::trusted_nets ===
+# A list of subnets to permit, in CIDR notation.
+#
+# If you need this to be more (or less) restrictive for a given class,
+# you can override it in Hiera.
+simp_options::trusted_nets:
+- 192.168.121.0/24
+
+# === sssd::domains ===
+# A list of domains for SSSD to use.
+#
+# * When `simp_options::ldap` is true, this field should include `LDAP`.
+# * When `simp_options::ldap` is false, this field must be a valid
+#   domain ('Local' and/or a custom domain) or the sssd service will
+#   fail to start.
+#
+sssd::domains:
+- LDAP
+
+# === svckill::mode ===
+# Strategy svckill should use when it encounters undeclared services.
+#
+# 'enforcing' = Shut down and disable all services not listed in your
+#               manifests or the exclusion file
+# 'warning'   = Only report what undeclared services should be shut
+#               down and disabled, without actually making the changes
+#               to the system
+#
+# NOTICE: svckill is the mechanism that SIMP uses to comply with the
+# requirement that no unauthorized services are running on your system.
+# Is it HIGHLY recommended that you set this to 'enforcing'. Please be
+# aware that, if you do this, svckill will stop ALL services that are
+# not referenced in your Puppet configuration.
+svckill::mode: enforcing
```

Comment:
This is good. We won't discover issues if it's set in `warning` mode.

Reply:
:skull_and_crossbones:  :+1: